core/src/main/java/io/jenkins/pluginhealth/scoring/scores/DocumentationScoring.java

```diff
@@ -57,26 +59,35 @@ public String description() {
 
     @Override
     public List<ScoringComponent> getComponents() {
-        return List.of(new ScoringComponent() {
-            @Override
-            public String getDescription() {
-                return "The plugin has a specific contributing guide.";
-            }
+        return List.of(
+            new ScoringComponent() {
+                @Override
+                public String getDescription() {
+                    return "The plugin has a specific contributing guide.";
+                }
 
-            @Override
-            public ScoringComponentResult getScore(Plugin plugin, Map<String, ProbeResult> probeResults) {
-                ProbeResult probeResult = probeResults.get(ContributingGuidelinesProbe.KEY);
-                if (probeResult != null && "Contributing guidelines found.".equals(probeResult.message())) {
-                    return new ScoringComponentResult(100, getWeight(), List.of("Plugin has a contributing guide."));
+                @Override
+                public ScoringComponentResult getScore(Plugin plugin, Map<String, ProbeResult> probeResults) {
+                    ProbeResult probeResult = probeResults.get(ContributingGuidelinesProbe.KEY);
+                    if (probeResult != null && "Contributing guidelines found.".equals(probeResult.message())) {
+                        return new ScoringComponentResult(100, getWeight(), List.of("Plugin has a contributing guide."));
+                    }
+                    return new ScoringComponentResult(
+                        0,
+                        getWeight(),
+                        List.of("The plugin relies on the global contributing guide."),
+                        List.of(new Resolution(
+                            "See why and how to add a contributing guide",
+                            "https://www.jenkins.io/doc/developer/tutorial-improve/add-a-contributing-guide/"
+                        ))
+                    );
                 }
-                return new ScoringComponentResult(0, getWeight(), List.of("The plugin relies on the global contributing guide."));
-            }
 
-            @Override
-            public int getWeight() {
-                return 2;
-            }
-        },
+                @Override
+                public int getWeight() {
+                    return 2;
+                }
+            },
```

Comment:
That is code indentation change only. Nothing specific to this pull request was done on the code.